thrall/app/lib/elasticsearch/ElasticSearch.scala

```diff
@@ -37,6 +37,10 @@ class ElasticSearch(config: ElasticSearchConfig, metrics: Option[ThrallMetrics])
     executeAndLog(getAliases(Nil, Seq(alias)), s"Looking up index for alias '$alias'").map(_.result.mappings.keys.headOption)
   }
 
+  def countDocsInIndex(index: Index)(implicit ex: ExecutionContext, logMarker: LogMarker = MarkerMap()): Future[Long] = {
+    executeAndLog(count(index.toIndexes), s"Counting docs in index ${index.name}").map(_.result.count)
+  }
+
```

Comment:
do the existing stat counts not provide the right info?

Reply:
Thanks for pointing me in the right direction - I hadn't realised that a count function already existed! I've pushed a new commit that makes use of that instead. It looks like it'll have a slight performance impact (it makes three queries for each index for different types of counts, of which we only use one here) but I think that's acceptable for powering this dashboard and avoiding duplication of logic